src/app/zap-templates/zcl/data-model/chip/door-lock-cluster.xml

```diff
@@ -14,159 +14,133 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-            <featureBit tag="COTA" bit="7">true</featureBit>
-            <featureBit tag="USR" bit="8">true</featureBit> <!-- TODO: Depends on ALIRO, [ PIN | RID | FGP | FACE ]-->
-            <featureBit tag="NOT" bit="9">false</featureBit>
-            <featureBit tag="YDSCH" bit="10">true</featureBit>
-            <featureBit tag="HDSCH" bit="11">true</featureBit>
-            <featureBit tag="UBOLT" bit="12">true</featureBit>
-            <featureBit tag="ALIRO" bit="13">true</featureBit>
-            <featureBit tag="ALBU" bit="14">true</featureBit> <!-- TODO: Depends on ALIRO -->
-        </globalAttribute>
-
-        <features>
-            <feature bit="0" code="PIN" name="PIN Credential" summary="Lock supports PIN credentials (via keypad, or over-the-air)">
-            <optionalConform/>
-            </feature>
-            <feature bit="1" code="RID" name="RFID Credential" summary="Lock supports RFID credentials">
-            <optionalConform/>
-            </feature>
-            <feature bit="2" code="FGP" name="FingerCredentials" summary="Lock supports finger related credentials (fingerprint, finger vein)">
-            <otherwiseConform>
-                <provisionalConform/>
-                <optionalConform/>
-            </otherwiseConform>
-            </feature>
-            <feature bit="3" code="LOG" name="Logging" summary="Lock supports local/on-lock logging when Events are not supported">
-            <optionalConform/>
-            </feature>
-            <feature bit="4" code="WDSCH" name="WeekDayAccessSchedules" summary="Lock supports week day user access schedules">
-            <optionalConform/>
-            </feature>
-            <feature bit="5" code="DPS" name="DoorPositionSensor" summary="Lock supports a door position sensor that indicates door's state">
-            <optionalConform/>
-            </feature>
-            <feature bit="6" code="FACE" name="FaceCredentials" summary="Lock supports face related credentials (face, iris, retina)">
-            <otherwiseConform>
-                <provisionalConform/>
-                <optionalConform/>
-            </otherwiseConform>
-            </feature>
-            <feature bit="7" code="COTA" name="CredentialsOverTheAirAccess" summary="PIN codes over-the-air supported for lock/unlock operations">
-            <optionalConform/>
-            </feature>
-            <feature bit="8" code="USR" name="User" summary="Lock supports the user commands and database">
-            <optionalConform>
-                <orTerm>
-                <feature name="PIN"/>
-                <feature name="RID"/>
-                <feature name="FGP"/>
-                <feature name="FACE"/>
-                </orTerm>
-            </optionalConform>
-            </feature>
-            <feature bit="9" code="NOT" name="Notification" summary="Operation and Programming Notifications">
-            <optionalConform/>
-            </feature>
```

Comment:
Enclosed in Zigbee conditionals in the spec definition, but its already included in the matter's xml version, so should be included here as well.
https://github.com/project-chip/connectedhomeip/blob/master/data_model/master/clusters/DoorLock.xml#L69

Reply:
Confirmed with @andreilitvin , matter must not include anything within the zigbee condititionals.